hardhat-solidity-tests/README.md

```diff
@@ -0,0 +1,10 @@
+Temporary directory to experiment with a Hardhat plugin that uses EDR's solidity tests feature.
+
+This directory can be removed once the plugin is handed over to the Hardhat team.
+
+## How to use
+
+1. Clone this repo and [the example](https://github.com/NomicFoundation/hardhat-solidity-tests-example)
```

Comment:
```suggestion
1. Clone this repo and [the usage example](https://github.com/NomicFoundation/hardhat-solidity-tests-example) to `./example`
```

Reply:
Note that the example repo is private, we should make that public probably

Reply:
No, that was my bad. There's no need to clone that repo anymore. I simplified the steps now.

Reply:
Ah ok cheers! Good to merge from my side